src/utils/input.js

```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
 
   return result
 }
+
+/**
+ * @param {number} valueLength - Length of input value
+ * @param {boolean} isRequired - If value is required
+ * @param {number} inputMaxLength - Minimum limit of length of the input value
```

Comment:
the parameter is named `max` but the comment is about `min`. Which one is correct?

Reply:
I'll rename it `expectedLength` as proposed below, because it's exactly that 👍